### ISSUE TITLE
How can I set global styles as well?

### ISSUE THREAD

**Issue by richard-lopes** [jxnblk/cxs#69]
I have been using styled-jsx but go interested to convert my little project to cxs.
So far, so good. It looks like everything is working but one thing.

I had the following global styles defined that I can't seem to be able to convert to cxs:
```
body::before {
    background-color: #5f00a8;
    background: linear-gradient(135deg, #5f00a8 35%, #ed005f 100%);
    content: '';
    display: block;
    height: 3px;

    position: fixed;
    top: 0;
    width: 100%;
    z-index: 1;
  }

  html, body, #root {
    height: 100%;
  }
```

Is there any way I can apply styles to the html and body tags of the container page ?

**Comment by asilvas** [comment]
You shouldn't need cxs for the above global styles.

**Comment by richard-lopes** [comment]
Thanks I did it another way.
I guess I found it handy in styled-jsx to also have my global styles in JS. Also was useful for theming.
But I am all good. I ported the whole project and it works just fine now. And I love the simplicity and elegance of your little library.

One thing I noticed before we close this. 
For CSS rules, properties that can take a number, if you give a number a `px` will be appended automatically. It does work with common properties likes `height`, `width`... But it adds unwanted `px` for `z-index` or `flex` for example.
We could document that a workaround to this  is to use strings instead of numbers like so for those properties: `flex: '1'` or `zIndex: '1'` to prevent a `px` to be appended.
